### PR TITLE
Partner Order Voters

### LINKED ISSUE #100: Limit visable order by partners
Partners should only see their orders.

### PR DESCRIPTION
**Pull request by akoebbe**
Closes #100 

This PR adds voters for partner orders as well as adds a filter to the repository if the user doesn't have VIEW_ALL rights for the orders. This uses the "Active partner" feature to make sure lists are limited to the partner the user has selected in the case of the user being part of multiple partners.

### CODE REVIEW

File: src/Security/BulkDistributionVoter.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        $order = $subject;
+
+        switch ($attribute) {
+            case self::VIEW:
+                return $this->canView($order, $user);
+            case self::EDIT:
+                return $this->canEdit($order, $user);
+        }
+
+        throw new \LogicException('This code should not be reached!');
```
Comment (ericpoe) [inline src/Security/BulkDistributionVoter.php:49]: This should probably be moved to the switch as the default case.

File: src/Security/PartnerOrderVoter.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        $order = $subject;
+
+        switch ($attribute) {
+            case self::VIEW:
+                return $this->canView($order, $user);
+            case self::EDIT:
+                return $this->canEdit($order, $user);
+        }
+
+        throw new \LogicException('This code should not be reached!');
```
Comment (ericpoe) [inline src/Security/PartnerOrderVoter.php:49]: This should probably be moved to the switch as the default case.

Comment (akoebbe) [reply src/Security/PartnerOrderVoter.php]: dfc7acf7eed256b220b218d7262378fca50e8146

File: src/Security/BulkDistributionVoter.php (hunk 1 above)
Comment (akoebbe) [reply src/Security/BulkDistributionVoter.php]: dfc7acf7eed256b220b218d7262378fca50e8146

**Pull request merged**